### ISSUE TITLE
Amazon Linux 2023 (EC2)

### ISSUE THREAD

**Issue by ddouglas** [bbc/audiowaveform#202]
Same vain as https://github.com/bbc/audiowaveform/issues/201 

Can we get a build that can be installed on AL2023? I'm fixing to put together a script that will build it from source like what the OP of the above thread did for their docker container. I'll post back for other to see if it works.

**Comment by chrisn** [comment]
I already provide debian packages for Amazon Linux 2 (see [releases](https://github.com/bbc/audiowaveform/releases)), I'd welcome a PR to add Dockerfile and build script for Amazon Linux 2023 - see https://github.com/bbc/audiowaveform/tree/master/rpm. Does this need to be a statically linked build?

**Comment by ddouglas** [comment]
So I opted to just try and build the container in Github Actions and extract the binary from the container and upload it to S3 so I can pull it down onto my EC2 instances when needed. That works and we've even gone a step further and just built a custom AMI with this binary embedded.

As far as the instructions for AL2, 9/10, they won't work, and this case they didn't, with Al2023 due to AWS removed EPEL


It was the following command that was failing for me.
```
sudo yum install -y epel-release
```

I'll spin up a new instance and get you better output. Just now seeing this and I was reporting back like I stated.
